app/views/claims/address.html.erb

```diff
@@ -3,50 +3,54 @@
     <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
     <%= form_for current_claim, url: claim_path do |form| %>
-      <fieldset class="govuk-fieldset">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-          <h1 class="govuk-fieldset__heading">
-            <%= t("tslr.questions.address") %>
-          </h1>
-        </legend>
-
-        <div class="govuk-form-group">
-          <%= form.label :address_line_1, class: "govuk-label" do %>
-            Building and street <span class="govuk-visually-hidden">line 1 of 2</span>
+      <%= form_group_tag current_claim do %>
```

Comment:
These outer groups weren't here before, and now we have nested form groups. Is that right? Also is it an issue that the outer group will get the errors class applied to it?

Reply:
Looking at this, the outer group is wrong, I've updated it, thank you.